app/Http/Controllers/AdminUserController.php

```diff
@@ -75,12 +75,13 @@ public function postEdit($id, Request $request)
         $adminModel = new Admin;
         $this->validate($request,
             [
-                'txtName' => 'required',
+                'txtName' => 'required|regex:/(^[A-Za-z ]+$)+/',
```

Comment:
Is txtName the name of a person? Be careful with this regex people can also have a "-" character in their name.